doc/api_assets/api-realtime-search.js

```diff
@@ -0,0 +1,106 @@
+var client = algoliasearch('E5KERA8ZVY', '488f2cfbe85e7d906ec1070b02a40a54')
+    index  = client.initIndex('NodeJSDoc_dev');
+
+var d = document,
+    searchIsInitialized = false,
+    searchWrapper = '#search-wrapper',
+    searchBox  = '#search',
+    cancelBtn  = '.search-wrapper__icon',
+    toc        = '#toc',
+    docContent = '#apicontent',
+    breadcrumb = '.type--left';
+
+var ac = autocomplete(searchBox, { hint: true, debug: true }, [
+  {
+    source: autocomplete.sources.hits(index, { hitsPerPage: 5 }),
+    templates: {
+      empty: function(data) {
+        return '<div class="no-results">We didn\'t find any results for "' + data.query + '". Sorry!</div>';
+      },
+      suggestion: function(suggestion) {
+        var stability = (suggestion.isStable) ? 'stable' : 'deprecated';
+        var showDescription = (suggestion._highlightResult.textValue.matchedWords.length) ? false : true;
+        var breadcrumb = suggestion.breadcrumb || '';
+
+        if (showDescription) {
```

Comment:
I think we could factorize those 2 blocks and just add the `description` tags+content if needed.